proxies/syntrest-proxy/config.yml

```diff
@@ -20,6 +20,8 @@ spec:
           cluster: dev-fss
         - application: testnav-generer-synt-amelding-service
           cluster: dev-gcp
+        - application: testnav-generer-arbeidsforhold-populasjon-service
```

Comment:
Hvorfor kan denne fjernes? Må ikke den være der for at `testnav-generer-arbeidsforhold-populasjon-service` skal kunne kalle på syntrest proxy?

Reply:
Fjernes? Jeg legger den til

Reply:
Du har helt rett, tror jeg trenger kaffe 😅